src/shared/test/has-own-test.js

```diff
@@ -0,0 +1,14 @@
+import { hasOwn } from '../has-own';
+
+describe('hasOwn', () => {
+  [
+    [{ foo: 'bar' }, 'foo', true],
+    [{ foo: 'bar' }, 'bar', false],
```

Comment:
Unless you're purposely illustrating that a property _value_ is not a property in `hasOwn`'s eyes, this test might be easier to read as
```suggestion
    [{ foo: 'bar' }, 'baz', false],
```

Reply:
I agree.